traefik/values.yaml

```diff
@@ -889,6 +889,11 @@ serviceAccountAnnotations: {}
 # -- [Resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for `traefik` container.
 resources: {}
 
+# -- Percentage of memory limit to set for GOMEMLIMIT
+# -- set as decimal (0.9 = 90%, 0.95 = 95% etc)
+# -- only takes effect when resources.limits.memory is set
+goMemLimitPercentage: 0.9
```

Comment:
@darkweaver87 @jnoordsij Would it make sense for you to move it as a child of `deployment:` ?

Reply:
Yeah that sounds reasonable. We could even do `resources` as it's also heavily tied to that, but I think `deployment` makes a little more sense.